torch/onnx/symbolic_opset9.py

```diff
@@ -335,40 +312,66 @@
 
 
 _INT64_MAX = 9223372036854775807
+_onnx_symbolic = functools.partial(registration.onnx_symbolic, opset=9)
+
+
+def _apply_params(*args, **kwargs):
+    """Returns a decorator that calls the decorated (higher-order) function with the given parameters."""
+
+    def _apply(fn):
+        return fn(*args, **kwargs)
+
+    return _apply
+
+
+def _export(name: str):
```

Comment:
@BowenBao Re: `return opset9.max_pool2d(  # type: ignore[attr-defined]`

The functions are no longer exactly defined but is still (only) called in symbolic_caffe2. I created this _export decorator to expose the names. When we have the dispatcher, we can leverage that instead.

Reply:
Should we add this to decorator for all those kinds to maintain same behavior?

What happens to `__all__`, should it be kept there if `_export` is added?

Reply:
I added `_export` to only those that are called internally. I am hoping to reduce the number of exported functions. The only risk is internal CI breakage that we don't know about.

if _export is added, the entry in `__all__` will be kept.

Reply:
Done